torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -17,7 +17,7 @@ namespace codegen {
 
 namespace {
 
-class CudaKernelGenerator : private kir::IrVisitor {
+class CudaKernelGenerator : private kir::ConstIrVisitor {
```

Comment:
if we keep the original IrVisitor and name the new one MutableIrVisitor we don't need these changes

Reply:
Renamed.